application/src/main/java/be/jpendel/application/PersonMapper.java

```diff
@@ -6,7 +6,9 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-public class PersonMapper {
+/* TODO : mark: PersonMapper en PersonDTO moeten volgens mij in de domain layer
+*/
+class PersonMapper {
```

Comment:
Voor je applicatie server kan een adapter staan die een xml, json, whatever datastructuur mapt naar een personDTO/ command. De applicatie service moet niet weten van die adapter. die specifieert in zijn contract gewoon: ik verwacht deze dto. Het is aan degene die de applicatie service aanroept om de dto aan te maken en te mappen. Meerdere adapters kunnen meerdere mappers hebben die allemaal naar een DTO mappen. 

De mapping die hier gebeurd is de vertaal slag van je domein entiteit naar je applicatie service contract dto en terug. Je applicatie service kent het domein, omgekeerd niet. Je entiteit naar je dto mappen kan enkel in je applicatie service. Maar een binnenkomende dto naar een domein entiteit of factory doorgegeven, daarvoor moet de dto in het domein gekend zijn. Want de factory van een domein entiteit hoort thuis in het domein.

Dus lang verhaal kort: ik ga akkoord met kris zijn korte uitleg :P